src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Commands/Command.cs

```diff
@@ -244,7 +244,7 @@ public void Execute(object parameter)
         public void Execute(TExecuteParameter parameter)
         {
 #pragma warning disable 4014
-            ExecuteAsync(parameter, false);
+            ExecuteAsync(parameter, false).Wait();
```

Comment:
.Execute is already blocking at is not marked as async so the .ExecuteAsync is called in sync - as far as I am understanding it all and if I am not wrong, just did a little test.